plugins/org.locationtech.udig.project.ui/schema/tool.exsd

```diff
@@ -436,6 +436,13 @@ See the command extension point
                </documentation>
             </annotation>
          </attribute>
+         <attribute name="toggeling" type="boolean" use="default" value="false">
```

Comment:
correct spelling would be toggling I guess

maybe `isToggleButton` would fit much better to `onToolbar` (see above)